test/tools/yulInterpreter/Interpreter.h

```diff
@@ -111,6 +111,11 @@ struct InterpreterState
 	/// Number of the current state instance, used for recursion protection
 	size_t numInstance = 0;
 
+	// KZG version hash
+	const util::FixedHash<1> KZGHashVersion = util::FixedHash<1>(1);
+	// KZG commitments
+	const std::array<u256, 2> KZGCommitments = {0x01, 0x02};
```

Comment:
KZG is just a type of commitment. Technically we could even get another KZG commitment for something else in a block in the future so I would not use it in the name. The fact that the commitments are for blobs is more relevant for disambiguation:

```suggestion
	// Blob commitment hash version
	util::FixedHash<1> const blobHashVersion = util::FixedHash<1>(1);
	// Blob commitments
	std::array<u256, 2> const blobCommitments = {0x01, 0x02};
```